cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/13-school-results.html

```diff
@@ -56,78 +56,131 @@ <h3 class="u-border-bottom"><span data-school-item="school">Example University</
             'This school is under Heightened Cash Monitoring by the U.S. Department of Education. <a href="https://studentaid.gov/data-center/school/htm" target="_blank" class="external-link" rel="noreferrer noopener">Find out more.</a>' | safe
         ) }}
 
-        <div class="hidden-undergrad-content" data-state-based-visibility="none">
-            <div class="block block__mid">
+        <div class="undergrad-content">
+            <div class="block block__mid school-results_comparison-section" data-status="max">
                 <h4>Graduation rate</h4>
                 <div class="school-results_rank-display">
-                    <div class="big-percentile"><span data-school-item="rateGraduation"></span></div>
-                    <div><span data-school-item="name"></span>'s graduation rate is <strong data-school-item="grad-rate_control"></strong>. If you are going to school to earn more money, you need to complete your degree to get that payoff.</div>
-                </div>
-                <div class="school-results_cohort-buttons">
-                    <p>Compare <span data-school-item="name"></span>'s rate to other 4 year schools that are also.:</p>
-                    <div>
-                        <button class="a-btn btn__cohort" data-button-cohort="">in the U.S.</button>
-                        <button class="a-btn btn__cohort" data-button-cohort="cohortRankByState">in [State]
-                        </button>
-                        <button class="a-btn btn__cohort" data-button-cohort="">[public/private]</button>
-                    </div>
+                    <div class="big-percentile"><span data-school-item="rateGraduation" data-number-display="percentage"></span></div>
+                    <div><span data-school-item="school"></span>'s graduation rate is <strong><span data-school-item="rateGraduation" data-number-display="percentage"></span></strong>. If you are going to school to earn more money, you need to complete your degree to get that payoff.</div>
                 </div>
+                <fieldset class="o-form_fieldset school-results_cohort-buttons">
+                    <legend>
+                        Compare <span data-school-item="school"></span>'s rate to other 4 year schools that are also:
+                    </legend>
+                    {{
+                        radio.render({
+                          'name':  "graduation-rate-meter-selector",
+                          'label': 'in the U.S.',
+                          'value': 'cohortRankByHighestDegree',
+                          'id': 'graduation-rate_us',
+                          'class': 'm-form-field__lg-target btn__cohort'
+                        })
+                    }}
+                    {{
+                        radio.render({
+                          'name':  "graduation-rate-meter-selector",
+                          'label': 'in <span data-school-item="stateName"></span>' | safe,
```

Comment:
Just noticed that the `stateName` value wraps to a new line. I think this should fix it:
```suggestion
                          'label': '<span>in <span data-school-item="stateName"></span></span>' | safe,
```

Reply:
Thank you! I was going to ask you about that bug today, but you beat me to it!